lib/modules/dosomething/dosomething_global/dosomething_global.features.language.inc

```diff
@@ -33,7 +33,7 @@ function dosomething_global_locale_default_languages() {
     'plurals' => 0,
     'formula' => '',
     'domain' => '',
-    'prefix' => 'es-mx',
+    'prefix' => 'mx',
```

Comment:
So we're talking about [this in Strongarm](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_global/dosomething_global.strongarm.inc#L17). My understanding is that this mapping is based on the language's "Language code", not the "Path prefix". The language code isn't editable (a good thing). See here:

![lang-code-prefix](https://cloud.githubusercontent.com/assets/149811/10033159/cfebf0ec-6155-11e5-9812-b654b09233fe.png)